src/Engine/System/SoundSystem.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+#ifndef _SOUND_SYSTEM_HPP_
+#define _SOUND_SYSTEM_HPP_
+
+#define HAVE_FMOD_LIBRARY false
+#include "System.hpp"
+#include "Component/SoundComponents/SoundComponent.hpp"
+
+#ifdef HAVE_FMOD_LIBRARY 
+#include "fmod.hpp"
+#include "fmod_studio.hpp"
+#endif
+
+class SoundComponent;
+
+class SoundSystem : public System {
+
+	friend Scene;
+	/* Attributes */
+	public:
+		std::vector<std::string> soundfiles = {
+			"drill.wav",
```

Comment:
We should add audio file names to the json parser

Reply:
Totally agree. This is a pretty clunky way to get sounds to play, but I just wanted something simple that works for now.